src/styles/blocks/variables-clock.scss

```
@@ -0,0 +1,13 @@
+$hour-hand-height: 333px;
+$minute-hand-height: 410px;
+$hands-width: 22px;
+$cercle-diametr: 30px;
+$clock-wrappers-size: ($minute-hand-height * 2);
+$hands-shift: ($cercle-diametr / 2 - $hands-width / 2);
+
+@mixin hands{
```

```suggestion
@mixin hands ($color, $width, $height, $duration) {
```

Why not to create a universal mixin that returns proper hand props?